packages/optimistic-oracle/test/keeper.js

```diff
@@ -0,0 +1,305 @@
+const winston = require("winston");
+const sinon = require("sinon");
+
+const { toWei, hexToUtf8, utf8ToHex } = web3.utils;
+
+const {
+  OptimisticOracleClient,
+  GasEstimator,
+  SpyTransport,
+  lastSpyLogLevel,
+  spyLogIncludes,
+  PriceFeedMockScaled
```

Comment:
I dont think you should be using the `PriceFeedMockScaled` anywhere within these tests like this. IMO this file should actually be deprecated as it's somewhat confusing and is not used anywhere else within the repo after the most recent refactors.

Ideally, this set of tests should just use normal feeds without any kind of scaling like this expected to be applied to them.

Reply:
Note that I changed the PriceFeedMockScaled code. It now copies directly from PriceFeedMock but allows you set the starting precision from a price like "10" --> 10e18, 10e6, etc.

I wanted to test that the proposer would use the same precision as the identifier calls for. For example, USDBTC would call for a price in 6 precision.

WDYT?

Reply:
ye, that makes sense. and to double-check this PriceFeedMockScaled was not used anywhere else, right?

Reply:
That's correct